tests/unit-tests/input/test_key_repeat_dispatcher.cpp

```diff
@@ -243,3 +251,14 @@ TEST_F(KeyRepeatDispatcherOnArale, repeat_for_regular_keys)
     dispatcher.dispatch(a_key_down_event());
     alarm_function();
 }
+
+TEST_F(KeyRepeatDispatcherOnArale, no_repeat_for_meta_keys)
```

Comment:
FWIW there's already `TestClientInputKeyRepeat.meta_keys_are_not_repeated_to_clients`

Reply:
Ah, so there is. I'll drop it if you want, but I don't imagine it does any harm to have a unit test as well.